components/suggestion/search_date_suggestion/index.ts

```diff
@@ -14,26 +14,30 @@ import {GlobalState} from 'types/store';
 
 import SearchDateSuggestion from './search_date_suggestion';
 
-function mapStateToProps(state: GlobalState) {
-    const currentUserId = getCurrentUserId(state);
-    const userTimezone = getUserTimezone(state, currentUserId);
-    const locale = getCurrentLocale(state);
+function makeMapStateToProps() {
+    const getUserTimezone = makeGetUserTimezone();
 
-    const enableTimezone = areTimezonesEnabledAndSupported(state);
+    return (state: GlobalState) => {
+        const currentUserId = getCurrentUserId(state);
+        const userTimezone = getUserTimezone(state, currentUserId);
```

Comment:
0/5, but it might be helpful to define a `getCurrentTimezone` since I imagine we'll need that more often than we would another user's timezone. It would also reduce the number of copies of this selector by a fair bit.